server.go

```diff
@@ -292,6 +298,55 @@ func newServer(listenAddrs []string, chanDB *channeldb.DB, cc *chainControl,
 		selfAddrs = append(selfAddrs, addr)
 	}
 
+	// If we were requested to route connections through Tor and to
+	// automatically create an onion service, we'll initiate our Tor
+	// controller and establish a connection to the Tor server.
+	//
+	// NOTE: v3 onion services cannot be created automatically yet. In the
+	// future, this will be expanded to do so.
+	if cfg.Tor.Active && cfg.Tor.V2 {
+		s.torController = tor.NewController(cfg.Tor.Control)
+		if err := s.torController.Start(); err != nil {
```

Comment:
Maybe this can be moved to the server's `Start` method?

Reply:
Fixed.